src/harbour-advanced-camera.cpp

```diff
@@ -48,9 +49,15 @@ int main(int argc, char *argv[])
 
     QQuickView *view = SailfishApp::createView();
 
+    ResourceHandler *handler = new ResourceHandler();
+    handler->acquire();
```

Comment:
do we need to release at any point? destruction?

Reply:
Yes we need to release when the up is minimized so you can adjust the volume.